go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -549,6 +550,21 @@ func (qre *QueryExecutor) execDDL(conn *StatefulConnection) (*sqltypes.Result, e
 		}
 	}
 
+	if conn == nil {
+		conn, err = qre.tsv.te.txPool.createConn(qre.ctx, qre.options, qre.setting)
+		if err != nil {
+			return nil, err
+		}
+		defer conn.Release(tx.ConnRelease)
+	}
+
+	// A DDL statement should commit the current transaction in the VTGate.
+	// The change was made in PR: https://github.com/vitessio/vitess/pull/14110 in v18.
```

Comment:
I think this link is supposed to be https://github.com/vitessio/vitess/pull/14110 ?